control/autoware_operation_mode_transition_manager/package.xml

```diff
@@ -1,5 +1,5 @@
 <package format="3">
-  <name>operation_mode_transition_manager</name>
+  <name>autoware_operation_mode_transition_manager</name>
   <version>0.1.0</version>
   <description>The vehicle_cmd_gate package</description>
```

Comment:
Not really related to this PR, but think the description is wrong.
Just to let you know. (I realized since I was modifying the vehicle_cmd_gate)

Reply:
I didn't notice it!
Thank you Sakayori-san. I'll just change them in this PR.